Source/CarthageKit/Xcode.swift

```diff
@@ -984,10 +1002,11 @@ public func buildScheme(scheme: String, withConfiguration configuration: String,
 					}
 
 			case 2:
-				let firstSDK = sdks[0]
-				let secondSDK = sdks[1]
+				let (simulatorSDKs, deviceSDKs) = SDK.splitSDKs(sdks)
+				let deviceSDK = deviceSDKs.first!
```

Comment:
It might be useful to generate an error here. Maybe
```swift
guard let deviceSDK = deviceSDKs.first else { fatalError("Could not find device SDK in \(sdks)") }
guard let simulatorSDK = simulatorSDKs.first else { fatalError("Could not find simulator SDK in \(sdks)") }
```